setup.py

```diff
@@ -51,6 +51,11 @@
             "face-mask-detector=face_mask_detector.command_line_interface:main"
         ]
     },
+    classifiers=[
```

Comment:
Nice job with adding the Python versions to the classifiers - I still need to learn how PyPI works with `setuptools`.

However I did find this:
https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

I'm betting that's what PyPI _actually_ uses for displaying the supported python versions. Could you please add:

```python
python_requires=">=3.6, <4"
```

At least I think that's what it should be - please double check me.

Also, here's an additional example: https://packaging.python.org/tutorials/packaging-projects/#creating-setup-py